### DISCUSSION TITLE
Mods don't run on She Will Punish Them Remastered

### DISCUSSION THREAD

**Discussion by FireMeshCloud** [BepInEx/BepInEx#292] [Help and Q&A]
### What happened?

Hi, I'm newer to mods, and I don't now if I'm the only newb on who that's fall, but I have installed BepInEx
Within SWPT, but nothing happens, any new folder in the folder "BepInEx", then any mods can be installed,
Please Help Me x_x"

### BepInEx Distribution

Stable from GitHub

### Environment

```markdown
- OS: Windows 10, latest version I think
- BepInEx: 5.4.14
- Game: She Will Punish Them Remastered
```

**Comment by ghorsington** [comment]
Greetings!

BepInEx appears to work just fine on She Will Punish Them:

![image](https://user-images.githubusercontent.com/13762037/128417799-09a9266e-0802-460c-8df5-f7af073d6cde.png)

As this is lacking further logs and information, I'm converting this issue into a discussion.

**Comment by ghorsington** [comment]
When having issues with BepInEx, I suggest following the [troubleshooting list](https://docs.bepinex.dev/articles/user_guide/troubleshooting.html). Most importantly,

1. [Make sure you use correct bitness for BepInEx](https://docs.bepinex.dev/articles/user_guide/troubleshooting.html#windows-check-the-bitness-of-the-game)
2. [Try enabling the console to see if any errors pop up](https://docs.bepinex.dev/articles/user_guide/troubleshooting.html#enable-console)

If the above did not help, can you please screenshot your game folder and BepInEx folder?

**Comment by FireMeshCloud** [comment]
Thanks for the vivid response, it's my fault if that's don't run, I have made a confusion, have a nice day and at the next time ;)

**Reply by FNYXx** [reply]
did you fix it?
Can you tell me if you fixed it?
